playbooks/group_vars/redis.yml

```diff
@@ -0,0 +1,3 @@
+# 2 x 3 GB=6 GB memory for Redis per node.
+# TODO: Automatically tune based on available memory.
```

Comment:
@dokterbob maybe create an issue for this? we can forgot about this easily

Reply:
Good point. https://github.com/ipfs-search/ipfs-search-deployment/issues/25